src/test/validation_chainstatemanager_tests.cpp

```diff
@@ -97,7 +98,13 @@ BOOST_AUTO_TEST_CASE(chainstatemanager)
     exp_tip = c1.m_chain.Tip();
     BOOST_CHECK_EQUAL(validated_tip, exp_tip);
 
-    // Avoid triggering the address sanitizer.
+    // Drop any events from the scheduler to avoid accessing memory that is going to be unloaded
+    m_node.scheduler->stop();
+    threadGroup.interrupt_all();
+    threadGroup.join_all();
+    GetMainSignals().FlushBackgroundCallbacks();
```

Comment:
Could be wrong, but I'd think FlushBackgroundCallbacks should be called before scheduler->stop because flushing uses the scheduler

Reply:
Never mind, I confused FlushBackgroundCallbacks with SyncWithValidationInterfaceQueue